docs/usage_guide.rst

```diff
@@ -881,16 +881,16 @@ For every evm address it is possible to trigger the process of detecting tokens
    :align: center
 
 
-EVM Transactions
+History events
 =====================
 
-Rotki is capable of getting and decoding (understand what is happening) your EVM (Ethereum Virtual Machine) transactions. Currently transactions are queried and decoded for Ethereum and Optimism. When you visit the ``EVM Transactions`` section the process to obtain all the information will start. You will be able to check the status in an informative breakdown per address. Free users are limited to the 100 latest transactions.
+Rotki is capable of getting and decoding (understand what is happening) your events (ethereum transactions, optimism transactions, ethereum staking, etc.). When you visit the ``History Events`` section the process to obtain all the information will start. You will be able to check the status in an informative breakdown per blockchain address. Free users are limited to the 100 latest events.
 
-.. image:: images/eth_tx_query_process.png
-   :alt: EVM transactions query status breakdown
+.. image:: images/events_query_process.png
+   :alt: History events query status breakdown
    :align: center
 
-It is possible that you need to redecode some transactions if the set of decoding rules is modified. To redecode a transaction you have two possibilities. The first of them is to click on the three dots to display the options for an EVM transaction and click on ``Redecode events``. This will start the process to read the transaction's events again and try to understand what happened in them.
+It is possible that you need to redecode some events. To redecode an evm transaction you have two possibilities. The first of them is to click on the three dots to display the options for an EVM transaction and click on ``Redecode events``. This will start the process to read the transaction's events again and try to understand what happened in them.
```

Comment:
you can only redecode transactions, the other events (non evm events) can't be redecode

Reply:
```suggestion
It is possible that you need to redecode events for an evm transaction. To do that you have two options. The first of them is to click on the three dots to display the options for an EVM transaction and click on ``Redecode events``. This will start the process to read the transaction's events again and try to understand what happened in them.
```

And what's the second option?

Reply:
> And what's the second option?

```
The second option is to redecode all the transactions in one page or all the transactions that have been queried. To do so you need to click on ``REDECODE EVENTS`` at the top of the page. This will allow to select between ``Only this page`` and ``All events``.
```